pkg/render/logstorage.go

```diff
@@ -1450,6 +1452,28 @@ func (es elasticsearchComponent) curatorCronJob() *batchv1.CronJob {
 
 	const schedule = "@hourly"
 
+	container := corev1.Container{
+		Name:            ESCuratorName,
+		Image:           es.curatorImage,
+		ImagePullPolicy: ImagePullPolicy(),
+		Env:             es.curatorEnvVars(),
+		LivenessProbe:   elasticCuratorLivenessProbe,
+		SecurityContext: securitycontext.NewNonRootContext(),
+		VolumeMounts:    es.cfg.TrustedBundle.VolumeMounts(es.SupportedOSType()),
+	}
+
+	_, esHost, esPort, _ := tigeraurl.ParseEndpoint(relasticsearch.GatewayEndpoint(es.SupportedOSType(), es.cfg.ClusterDomain, ElasticsearchNamespace))
+
+	envVars := []corev1.EnvVar{
```

Comment:
Why not put these inside `curatorEnvVars()`?

Reply:
Can do.